src/openapi.yaml

```diff
@@ -2070,6 +2289,11 @@ components:
           description: De naam van het kanaal (`KANAAL.naam`) waar het bericht op
             moet worden gepubliceerd.
           maxLength: 50
+        source:
```

Comment:
The example for this field seems to be `../dictionary`, not sure where that comes from?

<img width="295" height="234" alt="image" src="https://github.com/user-attachments/assets/d1148fca-5aa0-4224-b2e0-4791def21a5c" />